lang/python/CMakeLists.txt

```diff
@@ -77,6 +77,37 @@ if("${CMAKE_C_COMPILER_ID}" STREQUAL "Clang" OR "${CMAKE_C_COMPILER_ID}" STREQUA
         -Wno-int-conversion
         -Wno-sign-conversion
     )
+elseif("${CMAKE_C_COMPILER_ID}" STREQUAL "MSVC")
+    set(swig_c_flags
```

Comment:
Is there anyway to take this list from the strict defines that are above to avoid duplication? If its too messy or w/e I don't mind, just curious.

Reply:
Great point. I've re-evaluated and moved some of the compiler flags around. I've realized a number of those flags should be global C compiler settings rather than strict specific settings. 
I've updated a bunch of the flags to make them global. The swig flags now don't have much of an intersect with strict (but are still needed due to the way SWIG generates some of the code).